pom.xml

```diff
@@ -87,8 +87,8 @@
         <maven-spotless-plugin.version>2.27.2</maven-spotless-plugin.version>
         <maven-scalastyle-plugin.version>1.0.0</maven-scalastyle-plugin.version>
 
-        <iceberg.version>1.1.0</iceberg.version>
-        <hive.version>3.1.1</hive.version>
+        <iceberg.version>1.3.0</iceberg.version>
+        <hadoop.version>3.1.1</hadoop.version>
```

Comment:
Do you know if this is a typo?

Reply:
It was introduced by code merge. I supposed it should be:
1. `hive.version` should be removed
2. `hadoop.version` should be `3.2.2`

Maybe you can take a look at it. @baiyangtx

Reply:
Yes, it is introduced by code merge. I will fix it.